tools/cli/helpers/readJson.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import fs from 'fs';
+import chalk from 'chalk';
+
+// eslint-disable-next-line no-console
+const log = console.log;
+
+/**
+ * Reads a JSON file and return a parsed JSON object..
+ *
+ * @param {string} project - The project's name.
+ * @param {string} packageManager - Which package manager.
+ * @param {boolean} output - Should an information message be outputted.
+ *
+ * @returns {object} JS object of the json data.
+ */
+function readJson( project, packageManager, output ) {
+	const file = packageManager + '.json';
+	let data;
+	try {
+		data = fs.readFileSync( `projects/${ project }/${ file }`, 'utf8' );
+	} catch ( err ) {
+		output ? log( chalk.yellow( `This project does not have a ${ file } file.` ) ) : null;
+		return false;
```

Comment:
I note `false` is valid (if kind of useless) JSON. Would it make sense to return `undefined` instead to unambiguously indicate that no JSON was loaded?

Sure, callers will probably just check for falsiness anyway, but it's easy to allow a caller to make the differentiation if it wants to.

Reply:
Yeah edge case, but might as well be more specific now before it trips someone up in the future.

Reply:
21f3e2569